client/src/genetics/dice.ts

```diff
@@ -0,0 +1,42 @@
+export interface IDie {
+    index: number,
+    numeral: string,
+    symbol: string
+}
+
+export const DICE: IDie[] = [
+    {index: 0, numeral: "1", symbol: "⚀"},
+    {index: 1, numeral: "2", symbol: "⚁"},
+    {index: 2, numeral: "3", symbol: "⚂"},
+    {index: 3, numeral: "4", symbol: "⚃"},
+    {index: 4, numeral: "5", symbol: "⚄"},
+    {index: 5, numeral: "6", symbol: "⚅"},
+]
+
+const createDiceMap = (): { [key: string]: IDie; } => {
+    const map = {}
+    DICE.forEach(die => {
+        map[die.numeral] = die
+        map[die.symbol] = die
+    })
+    return map
+}
+
+export const DICE_MAP = createDiceMap()
```

Comment:
You can combine these into one i.e. 
`export const DICE_MAP = (function() => {...} )()`